src/containers/header/index.js

```diff
@@ -48,68 +74,73 @@ const Header = props => {
+          <input type="text" placeholder="search" />
+          <MdSearch className={styles.icon} />
+        </div>
+        {width < 768 && (
+          <div className={styles.menuIcons} onClick={checkMobileMenuStatus}>
+            {!isOpenMenu ? <FaBars /> : <FaWindowClose />}
           </div>
-          <div className={styles.menu}>
+        )}
+        <div id="menu" className={styles.menu}>
+          <div onClick={checkMobileMenuStatus}>
             <NavLink to="/sources" className={styles.link}>
               Sources
             </NavLink>
+          </div>
+          <div onClick={checkMobileMenuStatus}>
             <NavLink to="/categories" className={styles.link}>
               Categories
             </NavLink>
+          </div>
 
-            {isAuthenticated ? (
+          {isAuthenticated ? (
+            <div className={styles.buttonsBlock}>
               <button
                 type="button"
                 className={styles.button}
                 onClick={logOutRequest}>
                 Log out
               </button>
-            ) : (
-              <>
-                <button
-                  type="button"
-                  className={styles.button}
-                  onClick={openModalLog}>
-                  Log in
-                </button>
-                <button
-                  type="button"
-                  className={styles.button}
-                  onClick={openModalReg}>
-                  Register
-                </button>
-              </>
-            )}
-          </div>
-        </>
-      ) : (
-        <Burger
-          login={openModalLog}
-          register={openModalReg}
-          logout={logOutRequest}
-          isAuth={isAuthenticated}
-        />
-      )}
-      <Login 
-        modalStatus={isOpenLog} 
+            </div>
+          ) : (
+            <div className={styles.buttonsBlock}>
+              <button
+                type="button"
+                className={styles.button}
+                onClick={openModalLog}>
+                Log in
+              </button>
+              <button
+                type="button"
+                className={styles.button}
+                onClick={openModalReg}>
+                Register
+              </button>
+            </div>
+          )}
+        </div>
+      </>
+      <Login
+        modalStatus={isOpenLog}
         onModalClose={closeLoginModal}
         modalStatusForgotPass={isOpenForgotPass}
         onModalCloseForgotPass={setModalForgotPass}
       />
       <Register modalStatus={isOpenReg} onModalClose={closeRegModal} />
-      <ForgotPassword modalStatus={isOpenForgotPass} onModalClose={setModalForgotPass}/>
+      <ForgotPassword
+        modalStatus={isOpenForgotPass}
+        onModalClose={setModalForgotPass}
+      />
     </header>
   );
 };
 
 Header.propTypes = {
   logOutRequest: PropTypes.func.isRequired,
-  isAuthenticated: PropTypes.bool
+  isAuthenticated: PropTypes.bool,
```

Comment:
Make it required or add to code default values block

Reply:
done